prometheus-pushgateway/SOURCES/prometheus-pushgateway.init

```diff
@@ -29,7 +29,7 @@ kv[prog_name]="prometheus-pushgateway"
 kv.readSysconfig
 
 binary="${BINARY:-/usr/bin/prometheus-pushgateway}"
-user="${USER:-prometheus}"
+user="${USER:-prometheus-pushgateway}"
```

Comment:
I think it will be much better if a user will be just `pushgateway`, don't you think?

Reply:
Yes. This long-length name looks awful. Use `pushgateway` instead (c9d9525a6457dbbe6083cecc53da455ba9df145d).